packages/@lwc/engine-server/src/renderer.ts

```diff
@@ -84,6 +90,68 @@ export const renderer: Renderer<HostNode, HostElement> = {
         return (element.shadowRoot as any) as HostNode;
     },
 
+    getProperty(node, key) {
+        if (key in node) {
+            return (node as any)[key];
+        }
+
+        if (node.type === HostNodeType.Element) {
+            const attrName = getAttrNameFromPropName(key);
+
+            // Handle all the boolean properties.
+            if (isBooleanAttribute(attrName)) {
+                return this.getAttribute(node, attrName) ?? false;
```

Comment:
How about mirroring the browser behavior of getAttribute(), in fact even the engine-server/renderer's getAttribute() also returns `null`?
```suggestion
                return this.getAttribute(node, attrName) ?? null;
```

Reply:
If the attribute is a boolean attribute, the return value should be `false` get retrieving the property.